packages/autocomplete-shared/src/version.ts

```diff
@@ -1 +1 @@
-export const version = '1.9.2';
+export const version = '1.9.3';
```

Comment:
Good point, I saw that you updated manually these versions in the last `1.9.2` PR release.
Not sure why it stopped syncing these versions since `1.9.1` (`1.9.0` was fine)?

Reply:
IIRC there were some issues in the workflow when releasing 1.9.1 which didn't push to GitHub releases, which made us push a 1.9.2 immediately after.

I looked at the previous PR and indeed I manually updated the versions that were left intact to have them all in sync. I'll take care of this for the current PR as well.

Reply:
Thanks @dhayab!